doc/HttpLuaModule.wiki

```diff
@@ -3771,6 +3771,17 @@ or
     ngx.req.set_uri("/foo", true)
 </geshi>
 
+Starting from <code>0.10.16rc6</code> of this module, this function accepts an
+optional boolean <code>binary</code> argument to allow arbitrary binary URI
+data. By default, this <code>binary</code> argument is false and this function
+will throw out a Lua error such as the one below when the <code>uri</code>
+argument contains any control characters (ASCII Code 0 ~ 0x1F and 0x7F).
+
+<geshi lang="text">
+[error] 8069#8069: *1 unsafe byte "0xd" in uri "/foo\x0D\x0Abar", client: 127.0.0.1, server: localhost, request: "GET /t HTTP/1.1", host: "localhost"
+[error] 8069#8069: *1 lua entry thread aborted: runtime error: rewrite_by_lua(nginx.conf:43):3: attempt to set unsafe uri (maybe you want to set the 'binary' argument?)
```

Comment:
Addin indentation here is better so that our conversion tool can do better in the markdown output. Also as mentioned in another PR's discussion, we'd better avoid emitting two error log messages for a single issue. And we should avoid Lua API calls triggering error log entries without control (Lua exceptions could be caught to avoid the 2nd error log entry).